include/deal.II/lac/sparse_direct.h

```diff
@@ -48,6 +48,16 @@ namespace types
 #else
   using suitesparse_index = long int;
 #endif
+
+#ifdef DEAL_II_WITH_MUMPS
+  using mumps_index = MUMPS_INT;
+  using mumps_nnz   = MUMPS_INT8;
```

Comment:
MUMPS is such a mixed bag -- everyone calls the type `int64`, they call it `int8` :-)

(I imagine that's because they use the Fortran `INTEGER*8` type.)